### ISSUE TITLE
Dark Blue Theme Pro Blazor Data Grid Virtualize causes white grid header

### ISSUE THREAD

**Issue by Jeffbrus** [abpframework/abp#12438]
* *5.1*
* **User Interface** type Blazor
* EF Core

The grid header is when using the Virtualize option on the grid with the commercial theme Dark Blue

**Comment by Jeffbrus** [comment]
![image](https://user-images.githubusercontent.com/8357336/166809229-0443ea2a-1628-46dd-b6c0-d96b622b6415.png)

**Comment by enisn** [comment]
That style comes from Blazorise and it assumes that background will be always white:
![lepton-dark-blue-error](https://user-images.githubusercontent.com/23705418/212872632-65a49236-661b-4224-9956-555d7585da29.gif)

As a workaround you can override that style in your own application until Lepton theme covers that situation:

```css
.table-fixed-header > .table > thead:not(.table-thead-theme) > tr > th {
    background: transparent !important;
}
```

**Comment by enisn** [comment]
It's fixed and will be released in the next patch version of 7.0
